test/setup-beam.test.js

```diff
@@ -541,6 +541,26 @@ describe('.getOTPVersion(_) - Elixir', () => {
   })
 
   it('returns the expected value', async () => {
+    const previousRunnerArch = process.env.RUNNER_ARCH
+    process.env.RUNNER_ARCH = 'X64'
+
+    spec = '<26'
+    // assuming this won't be bumped, since it's out-of-support from Erlang/OTP
+    expected = 'OTP-25.3.2.21'
+    if (process.platform === 'win32') {
+      // because this is online and Windows doesn't have the OTP- prefix
+      expected = '25.3.2.21'
```

Comment:
For Windows, we get the results from the `releases` listing on GitHub so `OTP` is (for the time being) not there.

Lemme know if we need further tests in the `windows.yml`, though.